### DISCUSSION TITLE
Weird date in Nixvim flake

### DISCUSSION THREAD

**Discussion by refaelsh** [nix-community/nixvim#2289] [Q&A] [answered]
Whenever I do a `sudo nixos-rebuild switch --flake ~/repos/dotfiles/#myNixos --option eval-cache false`, I get the following output:
```
.
.
.
• Updated input 'nixvim':
    'path:/nix/store/g5dvas65d1h4h5jljh7qszh3kil3bys5-source/nixvim?lastModified=1&narHash=sha256-aBZc7rnbEwyRwhWWvMkKIXL2V2%2BpsSDrDOO/rSt61Do%3D' (1970-01-01)
  → 'path:/nix/store/malllcsmhchmi53y8waa6p0r1rv7qckv-source/nixvim?lastModified=1&narHash=sha256-aBZc7rnbEwyRwhWWvMkKIXL2V2%2BpsSDrDOO/rSt61Do%3D' (1970-01-01)
.
.
.
```

What's up with the weird date of `1970-01-01`? Did I stumble on a bug?

Thanks :-)

P.S.
Also, I wanted to ask question number 2: I get the `Updated input 'nixvim'` every time I rebuild my NixOS. Is that normal that I get this every time, in spite of the fact I changed nothing in my various flakes/nix files?

**Comment by siph** [comment]
>What's up with the weird date of 1970-01-01? Did I stumble on a bug?

That's the unix epoch: https://en.wikipedia.org/wiki/Unix_time

**Reply by refaelsh** [reply]
But why is so? Should it not be something like 21/09/2024 or something?

**Reply by HeitorAugustoLN** [reply]
This is the default of `lastModified` attribute for flake inputs. See https://nix.dev/manual/nix/2.22/command-ref/new-cli/nix3-flake.html?highlight=epoch#flake-reference-attributes

**Reply by refaelsh** [reply]
Then I am guessing that Nixvim's CI/CD does not update this field. Possible bug?

**Reply by HeitorAugustoLN** [reply, marked as answer]
No, it is due to your flake input being a local path without git. So it just uses the default value.

**Reply by refaelsh** [reply]
I understand. Thank you!

**Reply by HeitorAugustoLN** [reply]
No problem :)
